master/usage/ip-pool-manipulation.md

```diff
@@ -0,0 +1,54 @@
+---
+title: IP Pool Manipulation
+---
+This document describes how to make common modifications to Calico's
```

Comment:
Should this contain a brief preamble about what IP pools actually are?, i.e. what they are used for?